quadsim/robots/robot.py

```diff
@@ -1,44 +1,222 @@
-from typing import Any
-from typing import List
+"""Base class for all robots."""
+import numpy as np
+import pybullet_data
 
-# TODO: move Sensor and Motor to appropriate files
-
-
-class Sensor:
-    pass
-
-
-class Motor:
-    pass
+from quadsim.robots import motor_model
 
 
 class Robot:
     """Robot Base."""
 
-    def __init__(
-        self,
-        pybullet_client: Any,
-        motors: List[Motor],
-        sensors: List[Sensor],
-        on_rack: bool = False,
-    ) -> None:
+    def __init__(self, pybullet_client, config) -> None:
```

Comment:
I think my main issue is that it's difficult to tell what the signature of this class is on first glance. Generally, I'm not a big fan of defining a class (especially a base class) with an opaque single config object as its only argument.

I know that we will change this when we switch over to hydra. I'll help do that after this PR is merged =].

Reply:
Thanks. Once we switched over to hydra we will use the new paradigm.